packages/nimble-components/src/anchor/tests/anchor.stories.ts

```diff
@@ -0,0 +1,72 @@
+import type { Meta, StoryObj } from '@storybook/html';
+import { withXD } from 'storybook-addon-xd-designs';
+import { html } from '@microsoft/fast-element';
+import { createUserSelectedThemeStory } from '../../utilities/tests/storybook';
+import '../../all-components';
+import { AnchorAppearance } from '../types';
+import { bodyFont } from '../../theme-provider/design-tokens';
+import { hrefDescription } from '../../patterns/link/tests/doc-strings';
+
+interface AnchorArgs {
+    label: string;
+    href: string;
+    underlineVisible: boolean;
+    appearance: keyof typeof AnchorAppearance;
+}
+
+const metadata: Meta<AnchorArgs> = {
+    title: 'Anchor',
+    decorators: [withXD],
+    parameters: {
+        docs: {
+            description: {
+                component:
+                    'Per [W3C](https://w3c.github.io/aria-practices/#link), an anchor/link widget provides an interactive reference to a resource. The target resource can be either external or local, i.e., either outside or within the current page or application.'
+            }
+        },
+        design: {
+            artboardUrl:
+                'https://xd.adobe.com/view/33ffad4a-eb2c-4241-b8c5-ebfff1faf6f6-66ac/screen/bfadf499-caf5-4ca0-9814-e777fbae0d46/specs/'
+        },
+        actions: {}
+    },
+    // prettier-ignore
+    render: createUserSelectedThemeStory(html`
+        <style class='code-hide'>
+            * {
+                font: var(${bodyFont.cssCustomProperty});
+            }
+        </style>
+        Click on the <nimble-anchor
+            href=${x => (x.href !== '' ? x.href : null)}
+            ?underline-visible=${x => x.underlineVisible}
+            appearance=${x => x.appearance}
+        >
+            ${x => x.label}
+        </nimble-anchor> to navigate.
```

Comment:
Looks like the current story has some white space issue still: https://60e89457a987cf003efc0a5b-gryblvlhqq.chromatic.com/?path=/story/anchor--anchor&args=underlineVisible:true
![image](https://user-images.githubusercontent.com/1588923/206769079-49f354dc-40be-4f2f-9739-3f860117b8d1.png)

Reply:
This was an issue with my story. I had whitespace between the word "link" and the closing tag. The other test story was working properly.